lib/stdlib/doc/src/gen_server.xml

```diff
@@ -69,7 +69,9 @@ gen_server:abcast     -----> Module:handle_cast/2
 -                     -----> Module:code_change/3</pre>
 
     <p>If a callback function fails or returns a bad value, the
-      <c>gen_server</c> process terminates.</p>
+      <c>gen_server</c> process terminates. However, if a callback function
+      raises a <c>throw</c> exception, the callback will be treated as if
+      it succeeded and returned the exception's exit reason.</p>
```

Comment:
I wonder if that comes from ancient `catch` usage. I've seen throws being mis-labelled as normal returns elsewhere for that reason.